homeassistant/components/vizio/media_player.py

```diff
@@ -193,10 +224,10 @@ def set_volume_level(self, volume):
         """Set volume level."""
         if self._volume_level is not None:
             if volume > self._volume_level:
-                num = int(100*(volume - self._volume_level))
+                num = int(self._max_volume*(volume - self._volume_level))
```

Comment:
It is a good idea to clean up the code as  you go, so please add also whitespaces around the `*` operator.

Reply:
Fair, I will update this